src/database/standalone-sqlite.ts

```diff
@@ -1565,8 +1571,9 @@ export class StandaloneSqliteDatabaseWorker {
     if (bundledIn === undefined || Array.isArray(bundledIn)) {
       sqlParts.push(`SELECT * FROM (${itemsFinalSql})`);
     }
+
     sqlParts.push(
-      `ORDER BY 1 ${sqlSortOrder}, 2 ${sqlSortOrder}, 3 ${sqlSortOrder}`,
+      `ORDER BY 1 ${sqlSortOrder}, 2 ${sqlSortOrder}, 3 ${sqlSortOrder}, indexed_at ${sqlSortOrder}`,
```

Comment:
Let's change this to `4` instead of `indexed_at`. We don't need the field name for performance here, and I don't want us wondering why this is an exception to the pattern in the future.

Reply:
As @karlprieb pointed out on Slack, this also requires shuffling the field order in the inner queries.

Reply:
Done here [`f10117e` (#114)](https://github.com/ar-io/ar-io-node/pull/114/commits/f10117efd6371479afc4c6ed7c568382533a1993)